rust/src/composepost.rs

```diff
@@ -291,6 +292,34 @@ pub(crate) fn compose_postprocess(
     Ok(())
 }
 
+/// Given the contents of a /usr/lib/os-release file,
+/// update the `VERSION` and `PRETTY_VERSION` fields.
+pub(crate) fn mutate_os_release(contents: &str, base_version: &str, next_version: &str) -> String {
+    let mut buf = String::new();
+    for line in contents.lines() {
+        if line.is_empty() {
+            continue;
+        }
+        let prefixes = &["VERSION=", "PRETTY_VERSION="];
```

Comment:
`PRETTY_NAME`

Reply:
Oooh yes.  This is why we have code review...

Reply:
And now this is also tested by the `test-mutate-os-release.sh` too.